docker-bake.hcl

```diff
@@ -24,7 +24,13 @@ group "default" {
 		"vmclarity-cr-discovery-server",
 		"vmclarity-orchestrator",
 		"vmclarity-ui",
-		"vmclarity-ui-backend"
+		"vmclarity-ui-backend",
+	]
+}
+
+group "scanner-plugins" {
+	targets = [
+		"scanner-plugin-kics",
```

Comment:
I would add this target to the `default` group because I think this image will be used often for testing. We might as well built it with the other images